lib/matplotlib/artist.py

```diff
@@ -980,7 +980,12 @@ def update(self, props):
         ret = []
         with cbook._setattr_cm(self, eventson=False):
             for k, v in props.items():
-                k = k.lower()
+                if k != k.lower():
```

Comment:
> it will allow `arrow.set(patchA=foo)` which failed so far?

Yes, that will work, but give a confusing warning "Case-insensitive properties were deprecated ...", which you can silence with using `arrow.set(patcha=foo)`, which in turn will break after the deprecation.

can we make this

~~~
# excluding mixed case parameters from the deprecation
mixed_case_parameters = ['UVC', 'patchA', 'patchB']
if k != k.lower() and k not in mixed_case_parameters:
~~~

Reply:
But `set(patcha=foo)` didn't actually work before (and still fails with this patch).  The only difference is that `set(patchA=...)` used to fail and now will fail with a slightly more confusing error message, but will work after the deprecation.

Reply:
Well ok, While I think there must be a better solution, likely nobody will notice if this has been broken currently.